lighthouse-core/audits/manual/pwa-each-page-has-url.js

```diff
@@ -6,6 +6,17 @@
 'use strict';
 
 const ManualAudit = require('./manual-audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that suggests a site's pages should all have distinct URLs. */
```

Comment:
```suggestion
  /** Title of a Lighthouse audit that suggests each page in a web site should have a unique URL. */
```
should also follow the positive title format
(we should also probably standardize on "site" -> "web site", "page" -> "web page"...in many cases it's inferrable but not always)

Reply:
+1 we should standardize